docs/quickstart_guide.md

```diff
@@ -289,6 +343,24 @@ for benchmarking asynchronous (i.e., Knative Eventing) case and more details abo
 ### 3. Delete Deployed Functions
 **On the master node**, execute the following instructions below using **bash**:
 
+1. Delete **all** deployed functions:
+    ```bash
+    kn service delete --all
+    ```
+
+## V. Deploying eStargz-based Functions
+This section provides an example function run using a `nodejs` base image that has been converted to the `stargz` format. To create other images supported by `stargz`, please refer to the [creating-estargz-images-using-ctr-remote](https://github.com/containerd/stargz-snapshotter/tree/cmd/v0.12.1#creating-estargz-images-using-ctr-remote) section of the official `stargz` repository.
+### 1. Deploy and Invoke Functions
+**On the master node**, execute the following using **bash**:
+
+```bash
+kn service apply <name> -f <yaml_config_path> --concurrency-target 1
```

Comment:
Since you are referring to the nodejs image in the text above, I would provide the instantiated command here, i.e.
```bash
kn service apply stargz-test -f configs/knative_workloads/stargz-node.yaml --concurrency-target 1
```

Reply:
Thank you for this observation! I've updated the guide to exemplify the usage better.